src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -18,20 +20,36 @@ public class DeleteCommandParser implements Parser<DeleteCommand> {
      */
     public DeleteCommand parse(String args) throws ParseException {
         try {
-            String[] arr = args.split(",");
-            int size = arr.length;
-            for (int i = 0; i < size; i++) {
-                if (!DeleteCommandParser.isNumber(arr[i].trim())) {
+            String[] inputArr = args.split(",");
+            int inputSize = inputArr.length;
+            int[] intArr = new int[inputSize];
+
+            for (int i = 0; i < inputSize; i++) {
+                if (!DeleteCommandParser.isNumber(inputArr[i].trim())) {
                     throw new ParseException(
                             String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteCommand.MESSAGE_USAGE));
                 }
             }
-            Index[] indexes = new Index[size];
+            //convert string indexes to int indexes and sort int indexes in ascending order
+            for (int i = 0; i < inputSize; i++) {
+                intArr[i] = Integer.parseInt(inputArr[i].trim());
+            }
+            Arrays.sort(intArr);
```

Comment:
Does the indexes need to be sorted for the delete command to work properly?

Reply:
it will still delete accordingly without sorting but I sort so that the message printed abt each deleted person is in ascending order, wdyt abt this?

Reply:
Is the sorting necessary or should we preserve the order of the deletions?

Reply:
I thought it would be good to maintain the order of the priority as well in the case they delete a medium followed by high priority person. But if yall think its not necessary ill just delete the sort line.

Reply:
I feel that it is better to preserve the order of deletions so that the user ill not be disorientated or confused when they see like the order of message printed and the order of their deletion does not match.

Reply:
Noted ive removed the sort.